packages/client/hmi-client/src/components/sidebar-panel/documents-sidebar-panel.vue

```diff
@@ -50,21 +50,23 @@ const removeDocument = async (xddUri: string) => {
 	}
 	// remove the document from the project assets
 	if (resourcesStore.activeProject && resourcesStore.activeProjectAssets) {
-		const assetsType = PUBLICATIONS;
+		const assetsType = ProjectAssetTypes.PUBLICATIONS;
 		deleteAsset(resourcesStore.activeProject.id, assetsType, docAsset.id);
 		// remove also from the local cache
-		resourcesStore.activeProject.assets[PUBLICATIONS] = resourcesStore.activeProject.assets[
-			PUBLICATIONS
-		].filter((docId) => docId !== docAsset.id);
-		resourcesStore.activeProjectAssets[PUBLICATIONS] = resourcesStore.activeProjectAssets[
-			PUBLICATIONS
-		].filter((document) => document.id !== docAsset.id);
-		documents.value = resourcesStore.activeProjectAssets[PUBLICATIONS];
+		resourcesStore.activeProject.assets[ProjectAssetTypes.PUBLICATIONS] =
+			resourcesStore.activeProject.assets[ProjectAssetTypes.PUBLICATIONS].filter(
+				(docId) => docId !== docAsset.id
+			);
+		resourcesStore.activeProjectAssets[ProjectAssetTypes.PUBLICATIONS] =
+			resourcesStore.activeProjectAssets[ProjectAssetTypes.PUBLICATIONS].filter(
+				(document) => document.id !== docAsset.id
+			);
+		documents.value = resourcesStore.activeProjectAssets[ProjectAssetTypes.PUBLICATIONS];
 	}
 
 	// if the user deleted the currently selected document, then clear its content from the view
 	if (docAsset.xdd_uri === documentId.value) {
-		router.push('/docs'); // clear the doc ID as a URL param
+		router.push('/document'); // clear the doc ID as a URL param
```

Comment:
Using

```suggestion
	router.push({ name: RouteName.DocumentRoute }); // clear the doc ID as a URL param
```

would be less brittle. (Though please double check that it'll still clear the URL params, you may need to pass `params: {}` or something like that)

Reply:
Maybe this should be a different PR